[2022-06-21 14:38:07,751] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_23 manual__2022-06-21T14:38:00.740001+00:00 [queued]>
[2022-06-21 14:38:07,790] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_23 manual__2022-06-21T14:38:00.740001+00:00 [queued]>
[2022-06-21 14:38:07,790] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:38:07,791] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-21 14:38:07,791] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:38:07,822] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_23> on 2022-06-21 14:38:00.740001+00:00
[2022-06-21 14:38:07,830] {standard_task_runner.py:52} INFO - Started process 3036 to run task
[2022-06-21 14:38:07,837] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_23', 'manual__2022-06-21T14:38:00.740001+00:00', '--job-id', '784', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmpd6pcwxeq', '--error-file', '/tmp/tmpejxqm6n9']
[2022-06-21 14:38:07,840] {standard_task_runner.py:77} INFO - Job 784: Subtask START_COUNT_BY_DAY_23
[2022-06-21 14:38:08,043] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_23 manual__2022-06-21T14:38:00.740001+00:00 [running]> on host c087b85f9f15
[2022-06-21 14:38:08,215] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_23
AIRFLOW_CTX_EXECUTION_DATE=2022-06-21T14:38:00.740001+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-21T14:38:00.740001+00:00
[2022-06-21 14:38:08,262] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-21 14:38:08,277] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-23'}
[2022-06-21 14:38:08,703] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_23, execution_date=20220621T143800, start_date=20220621T143807, end_date=20220621T143808
[2022-06-21 14:38:08,733] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-21 14:38:09,592] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
